src/shared/lib/datetime.js

```diff
@@ -1,106 +1,146 @@
-/*
-  Get a date object offset for the current timezone
-*/
-export const getDate = function() {
-  const date = new Date();
-  const utcDate = new Date(date.toUTCString());
-  utcDate.setHours(utcDate.getHours() - 7);
-  return new Date(utcDate);
-};
+import { DateTimeFormatter, ZonedDateTime, LocalDateTime, LocalDate, ZoneId } from '@js-joda/core';
+import '@js-joda/timezone/dist/js-joda-timezone-10-year-range'; // minimize package size by only importing tz data for current year ±5 yrs
 
-/*
-  Get date formatted in YYYY-M-D
-*/
-export const getYYYYMD = function(date = getDate(), sep = '-') {
-  let localDate = date;
-  if (typeof localDate === 'string') {
-    localDate = new Date(localDate);
-  }
-  const month = localDate.getUTCMonth() + 1;
-  const day = localDate.getUTCDate();
-  const year = localDate.getUTCFullYear();
+// util functions
+
+const currentJsDate = () => new Date(Date.now()); // allows us to mock current date
+
+const currentZdt = () => ZonedDateTime.parse(currentJsDate().toISOString());
 
-  return `${year}${sep}${month}${sep}${day}`;
+const normalize = d => d.replace(/[\\/.]/g, '-'); // replaces slashes & dots with dashes
+
+// truncate ISO datetime to ISO date
+const truncate = datetime =>
+  datetime
```

Comment:
I recommend taking the first 10 characters. It's faster and more reliable.

Also, where is space in `2020-03-30T18:26:20.708516+02:00`?